src/bundle.ts

```diff
@@ -15,12 +15,9 @@ import {Transform} from 'stream';
 import File = require('vinyl');
 import * as logging from 'plylog';
 import {urlFromPath} from './path-transformers';
-import {StreamAnalyzer, DepsIndex} from './analyzer';
-import {compose} from './streams';
+import {StreamAnalyzer} from './analyzer';
```

Comment:
Consider enabling the two new noUnused* typescript 2.0 compiler options to catch this stuff.